### PR TITLE
Fix productization requirements in CSV

### PR DESCRIPTION
**Pull request by nizamial09**
Fixes the Bug 1784031

Signed-off-by: Nizamudeen <nia@redhat.com>

### CODE REVIEW

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -306,11 +306,11 @@ spec:
 
     ## Before Subscription
 
-    Before subcribing to RHOCS, there are two pre-requisites that need to be satisfied.
+    Before subcribing to OpenShift Container Storage, there are two pre-requisites that need to be satisfied.
```
Comment (mykaul) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: typo: subcribing -> subscribing

File: tools/csv-merger/csv-merger.go [hunk 2]
```diff
@@ -567,11 +567,11 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 
 ## Before Subscription
 
-Before subcribing to RHOCS, there are two pre-requisites that need to be satisfied.
+Before subcribing to OpenShift Container Storage, there are two pre-requisites that need to be satisfied.
```
Comment (mykaul) [inline tools/csv-merger/csv-merger.go]: Same typo as above

File: tools/csv-merger/csv-merger.go [hunk 3]
```diff
@@ -567,11 +567,11 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 
 ## Before Subscription
 
-Before subcribing to RHOCS, there are two pre-requisites that need to be satisfied.
+Before subscribing to OpenShift Container Storage, there are two pre-requisites that need to be satisfied.
 
 ### Namespace
 
-RHOCS runs only in the openshift-storage Namespace, which needs to be created before subscription. The following manifest can be used to create the Namespace.
+OpenShift Container Storage runs only in the openshift-storage Namespace, which needs to be created before subscription. The following manifest can be used to create the Namespace.
```
Comment (obnoxxx) [inline tools/csv-merger/csv-merger.go]:
```suggestion
OpenShift Container Storage runs only in the openshift-storage namespace, which needs to be created before subscription. The following manifest can be used to create the namespace.
```

**Comment by nizamial09** [comment]
Need to do testing

File: deploy/deploy-with-olm.yaml [hunk 4]
```diff
@@ -17,7 +17,7 @@ spec:
     metadata:
       creationTimestamp: null
   targetNamespaces:
-  - openshift-storage
+    openshift-storage
```
Comment (obnoxxx) [inline deploy/deploy-with-olm.yaml]: Why this hunk in the commit "Added the redhat logo on the operator hub" ?

Comment (obnoxxx) [reply deploy/deploy-with-olm.yaml]: This is even wrong and lets the test fail, I think:

```
uncommitted deploy/deploy-with-olm.yaml changes. run 'make gen-latest-deploy-yaml' and commit results.
make: *** [Makefile:94: verify-latest-deploy-yaml] Error 1
```

Comment (nizamial09) [reply deploy/deploy-with-olm.yaml]: I'll fix this in a moment.

**Comment by kshlm** [comment]
@nizamial09 You also need to add the icon to `ocsCatalog` in `pkg/deploy-manager/subscription.go` in the method generateClusterObjects. Then run `make gen-latest-deploy-yaml`.

**Pull request merged**